RunningOrder/Sprint/SprintList.swift

```diff
@@ -22,18 +22,22 @@ struct SprintList: View {
         List(selection: $selectedSprint) {
             Section(header: Text("Active Sprints")) {
                 ForEach(sprints) { sprint in
-                    Text(sprint.name)
-                        .tag(sprint)
-                }
-
-                Button(action: { self.showNewSprintModal.toggle() }) {
-                    Text("New Sprint")
+                    NavigationLink(
+                        destination: StoryList(stories: sprint.stories)
+                            .listStyle(PlainListStyle()),
+                        label: {
+                            Text(sprint.name)
+                        }).tag(sprint)
```

Comment:
what's the goal of the tag?

Reply:
Useless here indeed because `selectedSprint`is not used anymore

Reply:
Useless here indeed, because `selectedSprint` is not used anymore